ql/src/test/results/clientpositive/llap/timestamp_1.q.out

```diff
@@ -257,7 +329,7 @@ POSTHOOK: query: select cast(t as double) from timestamp_1 limit 1
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@timestamp_1
 #### A masked pattern was here ####
-1.2938436611E9
+1.293843661E9
```

Comment:
Does this precision values match with Hive 1.2?

Reply:
Yes the precision value is same.

`beeline -u "jdbc:hive2://zk0-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181,zk2-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181,zk4-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2"
select cast(cast('2011-01-01 01:01:01' as timestamp) as double) ;;
Connecting to jdbc:hive2://zk0-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181,zk2-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181,zk4-nikhil.ae4yqb3genuuvaozdfax4l12ud.dx.internal.cloudapp.net:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2
Connected to: Apache Hive (version 1.2.1000.2.6.5.3033-1)
Driver: Hive JDBC (version 1.2.1000.2.6.5.3033-1)
Transaction isolation: TRANSACTION_REPEATABLE_READ
Beeline version 1.2.1000.2.6.5.3033-1 by Apache Hive
0: jdbc:hive2://zk0-nikhil.ae4yqb3genuuvaozdf> select cast(cast('2011-01-01 01:01:01' as timestamp) as double) ;;
+----------------+--+
|      _c0       |
+----------------+--+
| 1.293843661E9  |
+----------------+--+
1 row selected (0.452 seconds)
0: jdbc:hive2://zk0-nikhil.ae4yqb3genuuvaozdf> select cast(cast('2011-01-01 01:01:01' as timestamp) as float) ;;
+---------------+--+
|      _c0      |
+---------------+--+
| 1.29384371E9  |
+---------------+--+
1 row selected (0.81 seconds)
0: jdbc:hive2://zk0-nikhil.ae4yqb3genuuvaozdf>`